fossunited/fossunited/utils.py

```
@@ -103,7 +116,101 @@ def make_badge(text="Default", size="sm"):
 		"Cancelled": ("#E74C3C", "#FFFFFF"),
 		"Default": ("#171717", "#FFFFFF"),
 	}
-
 	bg_color, text_color = colors.get(text, colors["Default"])
 
 	return f'<span class="badge badge-{size}" style="background-color: {bg_color}; color: {text_color};">{text}</span>'
+
+
+def get_doc_likes(doctype, name):
+	likes  = frappe.db.get_value(doctype, name, "_liked_by")
+	try:
+		# Parse the string into a list
+		likes = json.loads(likes)
+	except json.JSONDecodeError as e:
+		print(f"Error decoding JSON: {e}")
```

Maybe do frappe.throw, instead of silently ignoring...